docs/jazzy.yml

```diff
@@ -63,9 +63,6 @@ custom_categories:
       - NavigationMapView
       - NavigationMapViewDelegate
       - MapView
-      - AccountManager
-      - PointAnnotation
```

Comment:
These symbols don't exist in Nav SDK.

Reply:
Yep, these were previously extensions in the navigation SDK, but we’ve since removed the extensions.